daemon/web/components/Table/index.sass

```diff
@@ -0,0 +1,62 @@
+@import '../../styles/index'
```

Comment:
Is this import necessary, if it's already included in the main app?

Reply:
My current preference is to leave it in because I want to make the imports implicit. I'm being a bit defensive here, as I don't want any poorly-named CSS from other files conflicting with this one as the project scales.

Also, webpack won't bundle it if we leave it out.

What do you think?